index.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:ital,wght@0,300;0,400;0,600;1,300&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header>
+      <h1>Modern Designers Conference</h1>
+    </header>
+    <h2>August 6-8th, San Francisco, CA</h2>
+    <p class="call">To inspire designers everywhere to create a better world</p>
+    <nav>
+      <ul class="navigation">
+        <li><a href="#tickets">Tickets</a></li>
+        <li><a href="#schedule">Schedule</a></li>
+        <li><a href="#speakers">Speakers</a></li>
+        <li><a href="#talks">Talks</a></li>
+      </ul>
+    </nav>
+    <section id="tickets" class="tickets"></section>
```

Comment:
You have an extra closing `section` tag here on line 28.
```suggestion
    <section id="tickets" class="tickets">
```